autoscale/src/main/java/com/sequenceiq/periscope/converter/DistroXAutoscaleScalingActivityResponseConverter.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.periscope.converter;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.periscope.api.model.DistroXAutoscaleScalingActivityResponse;
+import com.sequenceiq.periscope.domain.ScalingActivity;
+
+@Component
+public class DistroXAutoscaleScalingActivityResponseConverter extends AbstractConverter<DistroXAutoscaleScalingActivityResponse, ScalingActivity> {
+
+    @Override
+    public DistroXAutoscaleScalingActivityResponse convert(ScalingActivity scalingActivity) {
```

Comment:
Please add a simple set of unit tests for this class

Reply:
Done